hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthTask.java

```diff
@@ -147,7 +147,7 @@ protected void runTask() throws Exception {
           unhealthyContainerStateStatsMap);
       long start = Time.monotonicNow();
       long currentTime = System.currentTimeMillis();
-      long existingCount = processExistingDBRecords(currentTime,
+      long existingCount = processExistingDBRecords(Time.monotonicNow(),
```

Comment:
After trace down the code, I think the `currentTime` param of `processExistingDBRecords` is not used for calculate duration, it just store it with `UnhealthyContainersRecord` in recon db.
```suggestion
      long existingCount = processExistingDBRecords(currentTime,
```

Reply:
@chiacyu Please revert both of the lines, not only `currentTime`.